source/a_box/tmu/tmu.c

```diff
@@ -188,49 +155,53 @@ void readTemps(tmu_handle_t *tmu) {
         tmu->tmu4_min = 0xFFFF;
     }
 
+    // skipping to measure power rail
+    if (curr_therm == NUM_THERM) {
+        curr_therm = 15;
+    }
 
     uint8_t therm = curr_therm;
     if (tmu_daq_override) therm = tmu_daq_therm;
+    //Select the MUX pin on each TMU board to read the thermistor value
     PHAL_writeGPIO(MUX_A_Port, MUX_A_Pin, (therm & 0x1));
     PHAL_writeGPIO(MUX_B_Port, MUX_B_Pin, (therm & 0x2));
     PHAL_writeGPIO(MUX_C_Port, MUX_C_Pin, (therm & 0x4));
     PHAL_writeGPIO(MUX_D_Port, MUX_D_Pin, (therm & 0x8));
-
 }
 
 
-void resistance_to_temp(float resistance, uint16_t *temp) {
-   float resistance_scaled = resistance / R25;
-    double native_log = native_log_computation(resistance_scaled);
-   if (COMP(resistance_scaled, LOW_RANGE_MIN, LOW_RANGE_MAX)) {
-       *temp = (uint16_t) 100 * ((1 / (LOW_RANGE_A + ((LOW_RANGE_B * native_log)) + (LOW_RANGE_C * native_log * native_log)
-               + (LOW_RANGE_D * native_log * native_log * native_log)) - 273.5));
-   }
-   else if (COMP(resistance_scaled, HIGH_RANGE_MIN, HIGH_RANGE_MAX)) {
-       *temp = (uint16_t) 100 * ((1 / (HIGH_RANGE_A + ((HIGH_RANGE_B * native_log) + (HIGH_RANGE_C * native_log * native_log)
-               + (HIGH_RANGE_D * native_log * native_log * native_log))) - 273.5));
-   }
-   else {
-       *temp = 0;
-   }
-}
+// void resistance_to_temp(float resistance, uint16_t *temp) {
+//    float resistance_scaled = resistance / R25;
+//     double native_log = native_log_computation(resistance_scaled);
+//    if (COMP(resistance_scaled, LOW_RANGE_MIN, LOW_RANGE_MAX)) {
+//        *temp = (uint16_t) 100 * ((1 / (LOW_RANGE_A + ((LOW_RANGE_B * native_log)) + (LOW_RANGE_C * native_log * native_log)
```

Comment:
remove old code, can always go back in version control to get it

Reply:
fixed